Source/Initialization/WarpXInitData.cpp

```diff
@@ -105,6 +105,169 @@ WarpX::PostProcessBaseGrids (BoxArray& ba0) const
+      amrex::Print()<<"                      | - div(B) cleaning\n";
+      }
+    if (do_multi_J == 1){
+        amrex::Print()<<"                      | - multi-J deposition\n";
+        amrex::Print()<<"                      |   - J-deposition number="<<WarpX::do_multi_J_n_depositions<<"\n";
+    }
+    if (fft_do_time_averaging == 1){
+      amrex::Print()<<"                      | - time-averaged \n";
+    }
+  #endif // WARPX_USE_PSATD
+
+  if (do_nodal==1){
+    amrex::Print()<<"                      | - nodal mode" <<"\n";
+  }
+  #ifdef WARPX_USE_PSATD
+    if ( (do_nodal==0) && (field_gathering_algo == GatheringAlgo::EnergyConserving) ){
+      amrex::Print()<<"                      | - staggered mode " <<"\n";
+    }
+    else if ( (do_nodal==0) && (field_gathering_algo == GatheringAlgo::MomentumConserving) ){
+      amrex::Print()<<"                      | - hybrid mode\n";
+      amrex::Print()<<"                      |   - field_centering_nox="<<WarpX::field_centering_nox<<"\n";
+      amrex::Print()<<"                      |   - field_centering_noy="<<WarpX::field_centering_noy<<"\n";
+      amrex::Print()<<"                      |   - field_centering_noy="<<WarpX::field_centering_noz<<"\n";
+      amrex::Print()<<"                      |   - current_centering_nox="<<WarpX::current_centering_nox<<"\n";
+      amrex::Print()<<"                      |   - current_centering_noy="<<WarpX::current_centering_noy<<"\n";
+      amrex::Print()<<"                      |   - current_centering_noy="<<WarpX::current_centering_noz<<"\n";
+    }
+    if (WarpX::use_hybrid_QED == true){
+      amrex::Print()<<"                      | - use_hybrid_QED = true" <<"\n";
+    }
+
+    // Print solver's order
+    amrex::Print()<< "Spectral order:       | - psatd.nox="<<nox_fft<<"\n";
+    amrex::Print()<< "                      | - psatd.noy="<<noy_fft<<"\n";
+    amrex::Print()<< "                      | - psatd.noz="<<noz_fft<<"\n";
+    if ((nox_fft==-1) or (noy_fft==-1) or (noz_fft==-1)) {
+      amrex::Print()<< "                      | ('-1' corresponds to infinite order); \n";
+    }
+    // Print guard cells number
+    amrex::Print()<< "Guard cells :         | - ng_alloc_EB="<<guard_cells.ng_alloc_EB<<"\n";
+    amrex::Print()<< "                      | - ng_depos_J="<<guard_cells.ng_depos_J<<"\n";
+    amrex::Print()<< "                      | - ng_depos_rho="<<guard_cells.ng_depos_rho<<"\n";
+    #endif // WARPX_USE_PSATD
+    amrex::Print()<<"-------------------------------------------------------------------------------"<<"\n";
+    //Print main boosted frame algorithm's parameters
+    if (WarpX::gamma_boost!=1){
+    amrex::Print()<<"Boosted Frame:        |    ON  "<<"\n";
+    amrex::Print()<<"                      |  - gamma_boost="<<WarpX::gamma_boost<<"\n";
+    amrex::Print()<<"                      |  - boost_direction=("<<WarpX::boost_direction[0]<<","<<WarpX::boost_direction[1]<<","<<WarpX::boost_direction[2]<<")\n";
```

Comment:
I think here there's some extra white space with respect to the indentation of the other parts, right? I would also put a space around the `=` sign, to make it more readable:
```suggestion
    amrex::Print()<<"Boosted Frame:        | ON  "<<"\n";
    amrex::Print()<<"                      | - gamma_boost = "<<WarpX::gamma_boost<<"\n";
    amrex::Print()<<"                      | - boost_direction = ("<<WarpX::boost_direction[0]<<","<<WarpX::boost_direction[1]<<","<<WarpX::boost_direction[2]<<")\n";
```